core/scripts/vrfv2/testnet/setup-envs/main.go

```diff
@@ -0,0 +1,448 @@
+		&nodes[scripts.VRFPrimaryNodeName].VrfKeys[0],
+		*linkAddress,
+		*linkEthAddress,
+		common.HexToAddress(*bhsContractAddressString),
+		common.HexToAddress(*batchBHSAddressString),
+		common.HexToAddress(*coordinatorAddressString),
+		common.HexToAddress(*batchCoordinatorAddressString),
+		&constants.MinConfs,
+		&constants.MaxGasLimit,
+		&constants.StalenessSeconds,
+		&constants.GasAfterPayment,
+		feeConfig,
+		nodes,
+	)
+
+	for key, node := range nodes {
+		client, app := connectToNode(&node.URL, output, node.CredsFile)
+
+		//GET ALL JOBS
+		jobIDs := getAllJobIDs(client, app, output)
+
+		//DELETE ALL EXISTING JOBS
+		for _, jobID := range jobIDs {
+			deleteJob(jobID, client, app, output)
+		}
+		//CREATE JOBS
+		switch key {
+		case scripts.VRFPrimaryNodeName:
+			createJob(jobSpecs.VRFPrimaryNode, client, app, output)
+		case scripts.VRFBackupNodeName:
+			createJob(jobSpecs.VRFBackupyNode, client, app, output)
+		case scripts.BHSNodeName:
+			createJob(jobSpecs.BHSNode, client, app, output)
+		case scripts.BHSBackupNodeName:
+			createJob(jobSpecs.BHSBackupNode, client, app, output)
+		case scripts.BHFNodeName:
+			createJob(jobSpecs.BHFNode, client, app, output)
+		}
+	}
+}
+
+func importVRFKeyToNodeIfSet(vrfBackupNodeURL *string, nodes map[string]scripts.Node, output *bytes.Buffer, file string) {
+	if *vrfBackupNodeURL != "" {
+		vrfBackupNode := nodes[scripts.VRFBackupNodeName]
+		vrfPrimaryNode := nodes[scripts.VRFBackupNodeName]
+
+		if len(vrfBackupNode.VrfKeys) == 0 || vrfPrimaryNode.VrfKeys[0] != vrfBackupNode.VrfKeys[0] {
+			client, app := connectToNode(&vrfBackupNode.URL, output, file)
+			importVRFKey(client, app, output)
+
+			vrfKeys := getVRFKeys(client, app, output)
+
+			vrfBackupNode.VrfKeys = mapVrfKeysToStringArr(vrfKeys)
+			if len(vrfBackupNode.VrfKeys) == 0 {
+				panic("VRF Key was not imported to VRF Backup Node")
+			}
+			printVRFKeyData(vrfKeys)
+		}
+	}
+}
+
+func getVRFKeys(client *clcmd.Shell, app *cli.App, output *bytes.Buffer) []presenters.VRFKeyResource {
+	var vrfKeys []presenters.VRFKeyResource
+
+	err := client.ListVRFKeys(&cli.Context{
+		App: app,
+	})
+	helpers.PanicErr(err)
+	helpers.PanicErr(json.Unmarshal(output.Bytes(), &vrfKeys))
+	output.Reset()
+	return vrfKeys
+}
+
+func createJob(jobSpec string, client *clcmd.Shell, app *cli.App, output *bytes.Buffer) {
+	if err := os.WriteFile("job-spec.toml", []byte(jobSpec), 0666); err != nil {
+		helpers.PanicErr(err)
+	}
+	job := presenters.JobResource{}
+	flagSet := flag.NewFlagSet("blah", flag.ExitOnError)
+	err := flagSet.Parse([]string{"./job-spec.toml"})
+	helpers.PanicErr(err)
+	err = client.CreateJob(cli.NewContext(app, flagSet, nil))
+	helpers.PanicErr(err)
+	helpers.PanicErr(json.Unmarshal(output.Bytes(), &job))
+	output.Reset()
+}
+
+func exportVRFKey(client *clcmd.Shell, app *cli.App, vrfKey presenters.VRFKeyResource, output *bytes.Buffer) {
+	if err := os.WriteFile("vrf-key-password.txt", []byte("twochains"), 0666); err != nil {
```

Comment:
And this?

Reply:
when exporting VRF Key, we need to specify a file where password is stored for import action